src/com/nutrons/steamworks/RobotBootstrapper.java

```diff
@@ -211,6 +214,15 @@ private Command hopperDrive(double distance1, double angle, double distance2) {
         Command.parallel(RobotBootstrapper.this.climbtake.pulse(true).delayFinish(500, TimeUnit.MILLISECONDS).then(RobotBootstrapper.this.climbtake.pulse(false).delayFinish(500, TimeUnit.MILLISECONDS)),
             Command.serial(RobotBootstrapper.this.drivetrain.driveDistance(distance1, 1, 10).endsWhen(Flowable.timer(1300, TimeUnit.MILLISECONDS), true),
                 RobotBootstrapper.this.drivetrain.turn(angle, 10),
-                RobotBootstrapper.this.drivetrain.driveDistance(distance2, 1, 10).endsWhen(Flowable.timer(1300, TimeUnit.MILLISECONDS), true)));
```

Comment:
Why did we move this? I think now we will only do drive -> turn, and only do the last drive while we are doing the other steps. Maybe it works tho, so I guess its ok. Only if we tested it on field.